scripts/microshift-cleanup-data.sh

```diff
@@ -56,7 +57,8 @@ function stop_clean_pods() {
             if [ "$(echo "${ocp_pods}" | wc -w)" -eq 0 ] ; then
                 break
             fi
-            crictl rmp -f "${ocp_pods}" &>/dev/null
+            # shellcheck disable=SC2086
+            crictl rmp -f ${ocp_pods} &>/dev/null
```

Comment:
This was a bug introduced when fixing shellcheck errors. The command expects multiple arguments, so quotes are not allowed.